src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj

```diff
@@ -96,20 +96,22 @@
   JARS_NO_REQUIRE is honored.  Setting both of those here for forward
   compatibility."
   [env :- jruby-schemas/EnvMap
```

Comment:
I think the docstring for this function needs to be updated to indicate arbitrary environment variables can be included when specified in the jruby config.